sunpy/map/tests/test_mapbase.py

```diff
@@ -194,7 +198,10 @@ def test_coordinate_frame(aia171_test_map):
     assert frame.observer.lon == aia171_test_map.observer_coordinate.frame.lon
     assert frame.observer.radius == aia171_test_map.observer_coordinate.frame.radius
     assert frame.obstime == aia171_test_map.date
-
+    
+def test_heliographic_longitude_crln(HMI_file):
+    if HMI_file.heliographic_longitude == 0:
```

Comment:
Why the if statement?

Reply:
To check if heliographic_longitude is none in the HMI file. Shouldn't it check?

Reply:
Would it be zero?

Reply:
`None` will be a better option?

Reply:
I mean I would remove it. I don't see why there should be an if statement.

Reply:
Okay, I will do that.